connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java

```diff
@@ -198,8 +198,7 @@ public Connector connector() {
     @Override
     public String toString() {
         return "WorkerConnector{" +
-                "connName='" + connName + '\'' +
-                ", connector=" + connector +
+                "id=" + connName +
```

Comment:
I see. I think I'll leave as is for now, it matches what we'd been doing. If we want to standardize logging format across the board (or figure out how to get structured log output!), I think we should figure that out collectively and select a structure that makes sense for everyone. Similarly, in various places I've started standardizing a bit on the `<variable>=<value>` at the tail end of log messages in the absence of a better structure.